bin/upstream-validator/index.ts

```diff
@@ -0,0 +1,40 @@
+const Ajv = require("ajv");
+const fs = require("fs").promises;
+
+async function main() {
```

Comment:
Would recommend the library we use in https://github.com/smart-on-fhir/health-cards/blob/main/generate-examples/src/index.ts#L2 (help output, formatting, etc)